test/CheckedC/static-checking/bounds-decl-checking.c

```diff
@@ -362,6 +379,15 @@ _Array_ptr<int> f55(unsigned num1, unsigned num2){
   return p;
 }
 
+_Array_ptr<int> test_f70(int c) : byte_count(c);
```

Comment:
Could you add some tests for _Nt_array_ptr too?

Reply:
@dtarditi Yes, I added some tests for _Nt_array_ptr and will push the changes shortly. There are more tests (for both nt_array_ptr and array_ptr) in [CheckedC repo](https://github.com/microsoft/checkedc/pull/368/files).